test/test_simulation_results.jl

```diff
@@ -864,21 +891,6 @@ function test_simulation_results_from_file(path::AbstractString, c_sys5_hy_ed, c
 
     @test_throws IS.InvalidValue set_system!(results_uc, c_sys5_hy_ed)
 
-    current_file = joinpath(
```

Comment:
The system will always be available in the simulation_store.h5 and so this condition should be impossible.

Reply:
Yeah, makes sense — @tengis-nrl sorry for making you write this test

Reply:
It did allow me to learn more about system files. No problem there.